docs/gaia/delegator-guide-cli.md

```diff
@@ -0,0 +1,405 @@
+
+::: warning
+*NOTE: To do this, **it is preferable to use a brand new ledger device.**. Indeed, there can be only one mnemonic per ledger device. If, however, you want to use a ledger that is already initialized with a seed, you can reset it by going in `Settings`>`Device`>`Reset All`. **Please note that this will wipe out the seed currently stored on the device. If you have not properly secured the associated mnemonic, you could lose your funds!!!***
+:::
+
+The following steps need to be performed on an un-initialized ledger device:
+
+1. Connect your ledger device to the computer via USB
+2. Press both buttons
+3. Do **NOT** choose the "Config as a new device" option. Instead, choose "Restore Configuration"
+4. Choose a PIN
+5. Choose the 12 words option
+6. Input each of the words you got during the fundraiser, in the correct order. 
+
+Your ledger is now correctly set up with your fundraiser mnemonic! Do not lose this mnemonic! If your ledger is compromised, you can always restore a new device again using the same mnemonic.
+
+Next, click [here](#using-a-ledger-device) to learn how to generate an account. 
+
+#### On a computer
+
+::: warning
+**NOTE: It is more secure to perform this action on an offline computer**
+::: 
+
+To restore an account using a fundraiser mnemonic and store the associated encrypted private key on a computer, use the following command:
+
+```bash
+gaiacli keys add <yourKeyName> --recover
+```
+
+You will be prompted to input a passphrase that is used to encrypt the private key of account `0` on disk. Each time you want to send a transaction, this password will be required. If you lose the password, you can always recover the private key with the mnemonic. 
+
+- `<yourKeyName>` is the name of the account. It is a reference to the account number used to derive the key pair from the mnemonic. You will use this name to identify your account when you want to send a transaction.
+- You can add the optional `--account` flag to specify the path (`0`, `1`, `2`, ...) you want to use to generate your account. By default, account `0` is generated. 
+
+### Creating an account
+
+To create an account, you just need to have `gaiacli` installed. Before creating it, you need to know where you intend to store and interract with your private keys. The best options are to store them in an offline dedicated computer or a ledger device. Storing them on your regular online computer involves more risk, since anyone who infiltrates your computer through the internet could exfiltrate your private keys and steal your funds.
+
+#### Using a ledger device
+
+When you initialize your ledger, a 24-word mnemonic is generated and stored in the device. This mnemonic is compatible with Cosmos and Cosmos accounts can be derived from it. Therefore, all you have to do is make your ledger compatible with `gaiacli`. To do so, you need to go through the following steps:
```

Comment:
It might be helpful here out of an abundance of caution to suggest that the Ledger device should be new, or it should be a device that the user trusts.